### PR TITLE
add 2.5 branch in github action

### PR DESCRIPTION
**Pull request by ldpliu**

### CODE REVIEW

File: .github/workflows/postsubmit.yml [hunk 1]
```diff
@@ -63,3 +63,41 @@ jobs:
...
+      - name: Deploy
+        uses: peaceiris/actions-gh-pages@v3
+        with:
+          publish_dir: ./clusterImageSets
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          publish_branch: release-2.5
+          keep_files: true
+          destination_dir: clusterImageSets
+
+  sync-clusterimageset-main:
```
Comment (qiujian16) [inline .github/workflows/postsubmit.yml:86]: why we need to update main. I think we should not push to main?

Comment (ldpliu) [reply .github/workflows/postsubmit.yml:86]: > why we need to update main. I think we should not push to main?

The 2.3 user still use the main branch, So we also need to sync the clusterimagesets in main branch.

Comment (qiujian16) [reply .github/workflows/postsubmit.yml:86]: I think we already update console chart to sub on 2.3 branch. For main branch. I'd rather we keep it static for backward compatible.

**Pull request merged**